addons/account_check_printing/__openerp__.py

```diff
@@ -10,7 +10,10 @@
 The check settings are located in the accounting journals configuration page.
     """,
     'website': 'https://www.odoo.com/page/accounting',
-    'depends' : ['account'],
+    'depends' : [
+        'account',
+        'account_voucher',  # Not for normal Odoo, but needed for migration
```

Comment:
@pedrobaeza I think nothing is realy forced, because account_voucher was a requirement for account_check_writing in 8.0. So if account_check_printing, the renamed account_check_writing is installed, it is certain account_voucher is also already installed. This is just to ensure that account_voucher is loaded (and migrated) before account_check_printing, because the migration of account_check_printing updates account_payment, that is filled by the migration of account_voucher.